### PR TITLE
Fix coverage reporting

### PR DESCRIPTION
**Pull request by nedbat**
This sets `relative_files=true`, and also uses a fix in coverage to make it all work.  I'll be releasing a new coverage version today, so take the parts of this you want, when you want them.

### Thanks for contributing, make sure you address all the checklists (for details on how see [development documentation](https://virtualenv.pypa.io/en/latest/development.html#development))

- [ ] ran the linter to address style issues (`tox -e fix_lint`)
- [ ] wrote descriptive pull request text
- [ ] ensured there are test(s) validating the fix
- [ ] added news fragment in `docs/changelog` folder
- [ ] updated/extended the documentation

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -51,8 +51,9 @@ description = [run locally after tests]: combine coverage data and create report
 skip_install = true
 deps =
     covdefaults>=2.2.2
-    coverage[toml]>=7.0.1
     diff_cover>=7.3
+    git+https://github.com/nedbat/coveragepy@nedbat/fix-virtualenv#egg=coverage==0.0
+    tomli
 extras =
```
Comment (gaborbernat) [inline tox.ini:57]: Why do we need this, isn't the `toml` extra already covering this?

Comment (nedbat) [reply tox.ini:57]: That was just for my debugging, I wasn't sure of the extra syntax with a git+ url, so I just stuck it in there like this.

Comment (nedbat) [reply tox.ini:57]: You can revert these lines. Coverage.py 7.0.2 is now out, with the fix.

**Comment by gaborbernat** [comment]
Moved to https://github.com/pypa/virtualenv/pull/2478

**Comment by gaborbernat** [comment]
Seems the wheel files are no longer excluded from coverage; see https://github.com/pypa/virtualenv/actions/runs/3824948711/jobs/6507540653. Do relative paths not work with `report.omit`?

**Pull request closed without merging**